### PR TITLE
Add pension email and mailing address

### PR DESCRIPTION
**Pull request by TaiWilkin**
## Summary
On the Pension Benefits application(21P-527EZ), moves the email/phone and mailing address sections to separate pages in Chapter 1 - Applicant Information, and updates the included fields for the 'Email address and phone numbers' page to match current designs. 

## How to run in local environment
1. Check out this branch locally
2. In your local environment, set the `pension_form_enabled` flipper to 'enabled' at http://localhost:3000/flipper/features/pension_form_enabled
3. Run `vets-website` and `vets-api`

## How to verify
1. Go to `http://localhost:3001/pension/application/527EZ` in your browser
2. Select 'Start your application without signing in'
3. Verify the page 'Mailing address' is in the correct location, per the designs, and includes only one 'Mailing address' title
4. Verify the page 'Mailing address' has a checkbox for "I receive mail outside of the United States on a U.S. military base" and has the dropdown for "Learn more about military base addresses"
5. Verify that when the checkbox for "I receive mail outside of the United States on a U.S. military base" is selected, the postal code must be a five-digit number.
6. Verify the page 'Email address and phone numbers' includes the fields email, phone number, mobile number, and international number. Email and mobile number should be required.

## Related issue(s)
[Ticket in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/70330)
[Ticket in GitHub](https://github.com/department-of-veterans-affairs/va.gov-team/issues/70329)
[Epic in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/63349)

## Testing done
- [x] Existing unit tests updated
- [x] New unit tests added

### Desktop
| Before | After (Mailing address) | After (Mailing address verification) | After (Phone and email) |  
| ------ | ------ | ------ | ------ |
|before | After | Screenshot 2023-11-29 at 3 13 12 PM | email-phone |

## What areas of the site does it impact?
Pension Benefits Application

## Acceptance criteria
[Ticket in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/70330)

### Quality Assurance & Testing
- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [x] Screenshot of the developed feature is added
- [x] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [x] Did you login to a local build and verify all authenticated routes work as expected with a test user

### CODE REVIEW

**Comment by mjknight50** [comment]
@mayacoforma A design comment..  it seems akward/weird to have Mailing Address listed twice.  Maybe there is some accessibility reason?

Screenshot 2023-11-29 at 2 45 42 PM

Otherwise, looks good.

**Review by mjknight50 — APPROVED** [review]

**Comment by TaiWilkin** [comment]
@mjknight50 That's an unfortunate side effect of the component being used - I replace it in [#26913](https://github.com/department-of-veterans-affairs/vets-website/pull/26913), which is built off of this branch, so it will be corrected as soon as that's it. We could merge that into this before proceeding with this PR, if you would prefer.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -74,12 +74,14 @@ src/applications/vre @department-of-veterans-affairs/benefits-non-disability @de
...
 src/applications/burials @department-of-veterans-affairs/benefits-non-disability @department-of-veterans-affairs/va-platform-cop-frontend
 src/applications/burial-poc-v6 @department-of-veterans-affairs/benefits-non-disability @department-of-veterans-affairs/va-platform-cop-frontend
-src/applications/pensions @department-of-veterans-affairs/benefits-non-disability @department-of-veterans-affairs/va-platform-cop-frontend
 src/applications/static-pages/vre-chapter-31 @department-of-veterans-affairs/benefits-non-disability @department-of-veterans-affairs/va-platform-cop-frontend
 src/applications/static-pages/vre-chapter-36 @department-of-veterans-affairs/benefits-non-disability @department-of-veterans-affairs/va-platform-cop-frontend
 src/applications/static-pages/coe-access @department-of-veterans-affairs/benefits-non-disability @department-of-veterans-affairs/va-platform-cop-frontend
 src/applications/sah @department-of-veterans-affairs/benefits-non-disability @department-of-veterans-affairs/va-platform-cop-frontend
 
+# Pensions
+src/applications/pensions @department-of-veterans-affairs/pensions @department-of-veterans-affairs/va-platform-cop-frontend
```
Comment (mjknight50) [inline .github/CODEOWNERS:83]: Good... yes, let's make this the standard for all front-end.

File: src/applications/pensions/config/form.js [hunk 2]
```diff
@@ -1,6 +1,5 @@
 import merge from 'lodash/merge';
 import get from 'platform/utilities/data/get';
```
Comment (mjknight50) [inline src/applications/pensions/config/form.js:2]: Importing from platform via platform/utilities is deprecated. Import from @department-of-veterans-affairs/platform-utilities

File: src/applications/pensions/config/form.js [hunk 3]
```diff
@@ -15,7 +14,6 @@ import bankAccountUI from 'platform/forms/definitions/bankAccount';
 import { VA_FORM_IDS } from 'platform/forms/constants';
 
 import currentOrPastDateUI from 'platform/forms-system/src/js/definitions/currentOrPastDate';
```
Comment (mjknight50) [inline src/applications/pensions/config/form.js:16]: There are a bunch of linting errors for deprecation of Importing from platform via platform/utilities

**Review by mjknight50 — CHANGES_REQUESTED** [review]
Can the linting errors be addressed?

**Review by ToddWebDev — APPROVED** [review]
Nice work, Tai! I'm getting one failing unit test locally, but it seems to be passing in CI.

**Review by mjknight50 — APPROVED** [review]

**Review by rmessina1010 — APPROVED** [review]
LGTM

**Pull request merged**